internal/view/assets/less/custom-dialog.less

```diff
@@ -153,4 +153,19 @@
 			}
 		}
 	}
-}
\ No newline at end of file
+}
+
+@media screen and (max-width: 400px) {
```

Comment:
I've made a quick test and maybe we only need this `@media` query for mobile devices:

```less
@media only screen and (max-width: 768px) {
	.custom-dialog-overlay {
		padding: 0px;
	}

	.custom-dialog {
		width: 100%;
		height: 100%;
	}
}
```

This makes the dialog use the entire viewport, which seems useful mobile side:

![Screenshot 2023-09-30 at 08 40 44](https://github.com/go-shiori/shiori/assets/812088/a6bfb930-c109-4383-bcaf-b4eecd0a38ac)

Don't get too distracted with mobile though, since there would be a lot of other things to improve in that side apart from this, just make it better without being to disruptive.